test/gtest/uct/test_uct_ep.cc

```diff
@@ -178,7 +181,10 @@ UCS_TEST_SKIP_COND_P(test_uct_ep, is_connected,
 
     disconnect();
     e1->destroy_ep(0);
-    e2->destroy_ep(0);
+
+    if (e2->iface_attr().cap.flags & UCT_IFACE_FLAG_CONNECT_TO_EP) {
```

Comment:
do we need to call destory_ep explicitly, now that we have add e1,e2 to m_entities?

Reply:
it should destroy:
https://github.com/openucx/ucx/blob/7ca7291132e22d784b5129dfbf957b6b5393774b/test/gtest/uct/uct_test.cc#L1278